WordPress/wporg-showcase-2022
# Update submission flow

Fixes #131. This uses the Jetpack contact form process originally created in #44, which is a change from the comment submission that the live site currently uses. That lets us easily add more fields using blocks that already exist, and the moderation process is moved out of comments into the "Feedback" section in wp-admin.

| Logged out | Form | Thanks |
|---|---|---|
| ![log-in](https://github.com/WordPress/wporg-showcase-2022/assets/541093/bc8813ce-2aaf-4bdd-a055-5322c13bd203) | ![form](https://github.com/WordPress/wporg-showcase-2022/assets/541093/97e1aa9e-e2ec-436c-a693-b42848f58132) | ![thanks](https://github.com/WordPress/wporg-showcase-2022/assets/541093/fbd9334a-3f48-44c4-af20-900fa2f0f0ea) |

To test:

1. Update the page content on the submit page (grab [from the staging site](https://wordpress.org/showcase-v2/wp-admin/post.php?post=91&action=edit) if needed)
2. Make sure the "Submission" template is used on the submit page
3. View the submission page while logged out
4. It should ask you to log in
5. Log in and try again
6. You should see the form
7. Submit a site
8. It should take you to the success page*
9. Your submission should be available on the backend of the site, under Feedback

*&nbsp;Currently the success redirect is not working. @StevenDufresne Did the form submission work correctly on the site last time? I'm wondering if it has something to do with using the form in a pattern. It reminds me of an issue I had in the past (https://github.com/Automattic/jetpack/pull/30764), but that was fixed and that root cause is not in place here.
